client/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+
+
+# Create your models here.
+class User(models.Model):
+    user_name = models.CharField(max_length=32, validators=[MinLengthValidator(1)], primary_key=True, null=False)
```

Comment:
min validator should be something like 6 too

Reply:
agree with you, I changed it